.goreleaser.yml

```diff
@@ -0,0 +1,81 @@
+project_name: centrifugo
+release:
+  github:
+    owner: centrifugal
+    name: centrifugo
+  name_template: '{{.Tag}}'
+  # If set to true, will not auto-publish the release.
+  draft: true
+  # If set to true, will mark the release as not ready for production.
+  prerelease: true
+brew:
+  commit_author:
+    name: Alexander Emelin
+    email: frvzmb@gmail.com
+  install: bin.install "centrifugo"
+  skip_upload: true
+builds:
+- goos:
+  - linux
+  - darwin
+  - windows
```

Comment:
Is it possible to build for exactly same platforms we already have in releases without windows 386? I.e:
* darwin amd64
* freebsd amd64
* linux 386
* linux amd64
* linux arm
* windows amd64

Reply:
No problem! Done!